### PR TITLE
[POAE7-2881]enable auto-vectorization for UOper，Constant

### PR DESCRIPTION
**Pull request by Ma-Jian1**
### What changes were proposed in this pull request?
enable auto-vectorization for UOper, Constant, and divide

### Why are the changes needed?
support auto-vectorization

### Does this PR introduce _any_ user-facing change?
No

### How was this patch tested?
benchmark test

### Which label does this PR belong to?
INFRA

### CODE REVIEW

File: cpp/src/cider/type/plan/BinaryExpr.cpp [hunk 1]
```diff
@@ -36,6 +36,7 @@ void BinOper::initAutoVectorizeFlag() {
       case kPLUS:
       case kMINUS:
       case kMULTIPLY:
+      case kDIVIDE:
```
Comment (bigPYJ1151) [inline cpp/src/cider/type/plan/BinaryExpr.cpp:39]: ```Div``` can't be vectorizable.

Comment (Ma-Jian1) [reply cpp/src/cider/type/plan/BinaryExpr.cpp:39]: enable Div for complex arithmetic operations, `i8*i16*2 + i16/3 - 1` gain 600%+ boost.
if we don't enable Div，`i8*i16*2 + i16/3 - 1` is slow than velox.

Comment (bigPYJ1151) [reply cpp/src/cider/type/plan/BinaryExpr.cpp:39]: Hm, I found div with literal is vectorizable, interesting.

Comment (Ma-Jian1) [reply cpp/src/cider/type/plan/BinaryExpr.cpp:39]: ok, I'll narrow auto-vectorization if the dividend is constant.

**Review by bigPYJ1151 — APPROVED** [review]

**Review by winningsix — APPROVED** [review]

**Pull request merged**